.golangci.yml

```diff
@@ -1,11 +1,61 @@
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    # - dogsled (TODO: write rational for disabling here, brought over from Makefile)
+    # - dupl (TODO: write rational for disabling here, brought over from Makefile)
+    - errcheck
+    - exhaustive
+    - funlen
+    # - gochecknoglobals (TODO: write rational for disabling here, brought over from Makefile)
+    # - gochecknoinits (TODO: write rational for disabling here, brought over from Makefile)
```

Comment:
I think the [CLI framework](https://github.com/spf13/cobra) we use requires them.